themes/default/content/blog/deploy-ai-ml-vercel-app/index.md

```diff
@@ -0,0 +1,207 @@
+If you are ready to integrate your chatbot and deploy it to Vercel along with us, then let's double check a few requirements before we continue. While Pulumi supports many languages, we chose Python for this project, as Python is a familiar language in the AI industry and this is part of the *#PulumiMLOps* challenge.
+
+### Requirements
+
+* [Pulumi CLI](https://www.pulumi.com/docs/install/)
+* [Pulumi account and access token](https://app.pulumi.com/signup)
+* [Python3](https://www.python.org/downloads/)
+* [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* Endpoint of Katwalk Server API. Follow the steps in [this article](https://www.pulumi.com/blog/mlops-the-ai-challenge-is-cloud-not-code/) to deploy the API and get your endpoint.
+* [Vercel account and access token](https://vercel.com/guides/how-do-i-use-a-vercel-api-access-token#creating-an-access-token)
+
+
+Now that you have everything needed to get started, let's hook up our frontend and deploy the frontend to Vercel using Pulumi.
+
+### 1. Clone the repo and setup your project
+
+The first thing we need to do is clone the repository from GitHub and setup the Pulumi Python environment.
+
+```bash
+# Clone the repository and cd to the infra directory
+git clone https://github.com/pulumiverse/katwalk && cd katwalk/pulumi
+
+# Create and initialize the python virtual env
+python3 -m venv venv && source venv/bin/activate
+
+# Install python dependencies
+python -m pip install -r requirements.txt
+```
+
+### 2. Login to Pulumi Cloud and initialize stack
+
+Next, let's setup our state and secrets store in Pulumi Cloud. 
+
+```bash
+# You can store Pulumi state in multiple ways, here we use Pulumi Cloud
+# Other state backends include Azure storage, Amazon S3, local file, and more
+pulumi login 
+
+# Initialize your stack
+# Here we name the stack "dev"
+pulumi stack init --stack dev
+```
+
+### 3. Configure deployment credentials
+
+Before the deployment, you will need to set the config values for your Vercel token, the repository name, and the repository type. The Vercel token is supposed to be set as a secret. If you open the `Pulumi.${stackName}.yaml` file, you can see the encrypted secret in this plain text YAML configuration.
+
+```bash
+# The name of the git repository. It should have this format {account-name}/{repository-name}
+pulumi config set gitRepoName <repo_name>
+
+# The git provider of the repository. Must be either `github`, `gitlab`, or `bitbucket`.
+pulumi config set gitRepoType <github|gitlab|bitbucket>
+
+# Configure vercel token as secret
+pulumi config set --secret vclToken <vercel_token>
+
+```
+
+### 4. Deploy to Vercel
+
+Now, you're set up to deploy your Vue app to Vercel. 
+
+```bash
+# Finally, run `pulumi up` to deploy!
+pulumi up
+```
+
+### 5. Interact with your chatbot
+
+Once your web app is running, you can visit the website, send prompts and see the response in real-time!
```

Comment:
you should specifically say that the URL for the website is in the outputs and you can visit it